core/src/main/resources/error/error-classes.json

```diff
@@ -46,9 +68,19 @@
     "message" : [ "Failed to rename %s to %s as destination already exists" ],
     "sqlState" : "22023"
   },
+  "FAILED_RENAME_TEMPORARY_FILE" : {
+    "message" : [ "Failed to rename temp file %s to %s as rename returned false" ]
+  },
   "FAILED_SET_ORIGINAL_PERMISSION_BACK" : {
     "message" : [ "Failed to set original permission %s back to the created path: %s. Exception: %s" ]
   },
+  "GET_PARTITION_METADATA_BY_FILTER" : {
```

Comment:
The naming of this isn't obviously an error. Can we name this something like `CANNOT_GET_PARTITION_METADATA_BY_FILTER`?